### PR TITLE
dev-libs/libxslt: Fix autoconf error when upgrading.

### PR DESCRIPTION
**Pull request by Allen-Webb**
Signed-off-by: Allen Webb <allenwebb@google.com>

### CODE REVIEW

File: dev-libs/libxslt/libxslt-1.1.34.ebuild [hunk 1]
```diff
@@ -19,6 +19,8 @@ KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~riscv ~
 IUSE="crypt debug examples python static-libs elibc_Darwin"
 REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
 
+BDEPEND=">=dev-util/pkgconfig-0.29.2"
```
Comment (mattst88) [inline dev-libs/libxslt/libxslt-1.1.34.ebuild:22]: No no no no no :)

There's `virtual/pkgconfig` that packages should depend on.

But, more to the point, if a patch claims to fix an error, it should describe what that error is.

File: dev-libs/libxslt/libxslt-1.1.34.ebuild [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
```
Comment (mattst88) [inline dev-libs/libxslt/libxslt-1.1.34.ebuild:4]: Let's not do this in this commit.

File: dev-libs/libxslt/libxslt-1.1.34.ebuild (hunk 1 above)
Comment (mattst88) [reply dev-libs/libxslt/libxslt-1.1.34.ebuild:22]: Oh, but I see you're adding BDEPEND. Okay.

Comment (Allen-Webb) [reply dev-libs/libxslt/libxslt-1.1.34.ebuild]: I changed it to a BDEPEND on virtual/pkgconfig with a hard blocker on the versions that don't work.

File: dev-libs/libxslt/libxslt-1.1.34.ebuild (hunk 2 above)
Comment (Allen-Webb) [reply dev-libs/libxslt/libxslt-1.1.34.ebuild:4]: Isn't EAPI=7 needed for BDEPEND?

Comment (leio) [reply dev-libs/libxslt/libxslt-1.1.34.ebuild:4]: Yes, but for that reason you wouldn't do this in BDEPEND, but EAPI-6 DEPEND

File: dev-libs/libxslt/libxslt-1.1.34.ebuild (hunk 1 above)
Comment (leio) [reply dev-libs/libxslt/libxslt-1.1.34.ebuild]: And what about pkgconf?

**Comment by thesamesam** [comment]
https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=61cb3f7f415df49e84a45b1f9803d15378e1d3da

**Pull request closed without merging**